jykuo-love-shiritori/twp
# [frontend] Info Settings Section

Reeeeesolves #13 partially

### Changes
### (D) Info Settings
- [x] [/user/info](http://localhost:5173/user/info)
  - navigate steps: person icon (nav bar) > Personal Info
  - this is made by @KatoToyoki 
- [x] [/seller/info](http://localhost:5173/user/seller/info)
  -  navigate steps: Sell (nav bar) > Shop Info
 
### Testing Approach (if unconventional)


- start the frontend server
  ```sh
  cd frontend
  npm run dev
  ```
- go to the new pages via steps above
- check and click every pixel if u want

### Additional context


- things haven't been implemented 
  - read and display the initial values of each field
  - make the json format of the forms
